### PR TITLE
Clean policy

### PR DESCRIPTION
**Pull request by soltysh**
This is one of 1.8 post-rebase tasks. It includes:

- reformatting so it's easier to compare with upstream roles
- removes networkpolicies from extensions, since they moved to own group. What worries me here is that I don't see any networkpolicy-related role in upstream, so I'm not sure if we might be bitten by it  @openshift/sig-networking any ideas why we had read/write access for extensions.networkpolicies?

@deads2k || @liggitt ptal
/cc @openshift/sig-security

### CODE REVIEW

**Comment by simo5** [comment]
I see other changes besides networkpolicies, but it is hard to follow whatthey are due to the mixing of reformatting and other changes.
for review it would be much better to split commits so that each intentional change is in its own commit with a comment that says why and what is being done.

**Comment by danwinship** [comment]
> removes networkpolicies from extensions, since they moved to own group.

Is that correct, compatibility-wise? In 1.8/3.8 extensions.NetworkPolicy is still the "default" type. (eg, if you "oc get networkpolicies", you get back extensions.NetworkPolicies, not networking.NetworkPolicies)

> What worries me here is that I don't see any networkpolicy-related role in upstream, so I'm not sure if we might be bitten by it

"admin" should definitely have read/write access to NetworkPolicy. I guess we didn't add that in OpenShift until after all the role stuff got upstreamed, but I was still thinking of roles as being OpenShift-only at that point so it didn't occur to me to fix it upstream too. I can do a PR once we figure out for sure if it's just networking or both networking and extensions.

**Comment by enj** [comment]
Any powers that you intend to actually remove need to be added to the dead.go files.
